third_party/3/pyspark/serializers.pyi

```diff
@@ -16,7 +16,13 @@
 # specific language governing permissions and limitations
 # under the License.
 
-from typing import Any, Optional
+from typing import Any, Dict
+
+basestring = str
+unicode = str
+xrange = range
```

Comment:
I think this should go away.

Reply:
I agree, somewhere conflict resolution went wrong